db/migrate/20190312104101_add_user_to_case.rb

```diff
@@ -0,0 +1,15 @@
+class AddUserToCase < ActiveRecord::Migration[5.0]
+  # To allow Case reports to show which user created the case.
+  # For existing cases, a 'dummy' user must be created with a fixed ID.
+  # The user creation is done here rather than data_migrations to ensure
+  # the DefaultUser exists before the column is created.
+  def up
+    DefaultUser.build!
```

Comment:
As the default user is only used during the migration, wouldn't it be better for it to be in here? Actually I don't think we need a default user class at all - simply create a user in the migration, save with validation: false and we're done.